templates/contribute/style.md

```diff
@@ -20,6 +20,9 @@ rather than rigid rules.
 - `m`, `n`, `k`, ... for natural numbers
 - `i`, `j`, `k`, ... for integers
 
+Types with a mathematical content are expressed with the usual
+mathematical notation, often with an upper case letter 
+(`G` for a group, `R` for a ring, `K` or `𝕜` for a field, `E` for a vector space, ...)
```

Comment:
Say that earlier we used `α`, `β` for groups as well, and PRs renaming all variables in a file are welcome?

Reply:
Done